src/main/java/com/google/idea/perf/tracer/ui/TracerPanel.kt

```diff
@@ -83,7 +83,7 @@ class TracerPanel(
     private var uiOverhead = 0L
 
     companion object {
-        private const val REFRESH_DELAY_MS = 30L
+        private const val REFRESH_DELAY_MS = 100L
```

Comment:
I am sort of fond of the smooth 30-fps refresh rate; and it seems to work well in most use cases. What are your thoughts?

(If UI overhead is a concern, one idea is to adapt the refresh rate based on the workload, e.g. ensure that UI overhead never exceeds X%.)

Reply:
I didn't like seeing large UI overheads initially and I didn't think that this was intended to be near real time(prefer form over function).
I did some experiments locally and I kinda get the annoyance of 300ms refresh rate. An interesting note: setting it to 60fps actually makes things worse. Because it makes it harder to register values with bare eyes.
Overall: 30ms-50ms seems pretty smooth, but I assume it's not worth changing for a small amount.
PS I wish there was a button to go to the precision mode